README.md

```diff
@@ -387,7 +387,9 @@ if (process.env.NODE_ENV === 'production') {
 // Then in your <head>
 {css}
 ```
+### CSS modules
 
+[Css modules](https://github.com/css-modules/css-modules) are support by default for all files with `.module.(css|less|scss|sass)` extension.
```

Comment:
Well that would be great too (adding to the starter) :-) but I meant here to add a simple example like:

```javascript
// Uses CSS Modules
import './my-component.module.css'

// Doesn't use CSS Modules
import './main.css'
```